leetcode/src/2279.c

```diff
@@ -0,0 +1,29 @@
+int compare(const int* i, const int* j)
+{
+    return *i - *j;
+}
+
+// Sorting.
+// Runtime: O(n*log(n))
+// Space: O(n)
+int maximumBags(int* capacity, int capacitySize, int* rocks, int rocksSize, int additionalRocks){

```

Comment:
Non-blocking, but makes it look better, IMO (same in other parts of the code).
```suggestion
int maximumBags(int* capacity, int capacitySize, int* rocks, int rocksSize, int additionalRocks) {
```